tiledb/sm/compressors/CMakeLists.txt

```diff
@@ -60,6 +60,33 @@ target_link_libraries(
     tdb_gzip_embedded_data PUBLIC 
     filter # OBJECTS library
     )
+    
+#
+# link test for some webp routines
+#
+find_package(Webp_EP REQUIRED)
+add_executable(link_webp_test1)
```

Comment:
Should we try running this?

Reply:
Not particularly functional but now running simply displaying 'shorthelp' output showing executable viability.